lib/index.js

```diff
@@ -27,7 +37,7 @@ exports.Service = class Service {
 
     get context() {
 
-        return this.server.realm.settings.bind || null;
+        return Hoek.reach(this, 'server.realm.settings.bind') || null;
```

Comment:
If the bound instance is computed lazily (only once `bind()` is called) then this can go back to the way it was and the first test wont fail.  In a _perfect_ world, computing the bound instance wouldn't actually cause any of the getters to be evaluated.